pages/whats-new/product-roadmap.md

```diff
@@ -16,10 +16,54 @@ Below is our product roadmap: a long-term plan of the goals, features, and long-
 <!-- TODO: Make these into issues; add roadmap project board
 You can also <a href="https://github.com/uswds/uswds/milestone/52" class="">view our product roadmap goals on GitHub</a>. -->
 
-<h2>2020 Roadmap</h2>
+<h2>Product Roadmap</h2>
 
 <div class="maxw-tablet">
   <ul class="site-roadmap-list">
+    <li class="site-roadmap-list__item"><span class="flex-fill">Value and benefits</span>
```

Comment:
We should change these headers to `flex-auto` or `flex-column`. IE11 seems to have trouble with it:

![image](https://user-images.githubusercontent.com/3385219/108372447-74adad80-71c4-11eb-821a-11dbed4ed4d0.png)

Reply:
Resolved in 0679bab